Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs

```diff
@@ -344,12 +347,13 @@ private void OnRoundStart(EntityUid uid, NukeopsRuleComponent? component = null)
             return;
 
         component.TargetStation = _random.Pick(eligible);
+        component.OperationName = _randomMetadata.GetRandomFromSegments(new List<string> {"operationPrefix", "operationSuffix"}, " ");
```

Comment:
add constant prototype validators at the top of the file.

these would look like:
```cs
    [ValidatePrototypeId<DatasetPrototype>]
    private const string OperationPrefixDataset = "operationPrefix";
```

Reply:
perhaps just
```cs
private const List<ProtoId<DatasetPrototype>> OperationNamesDatasets = new {"operationPrefix", "operationSuffix"};

Reply:
> perhaps just
> 
> ```cs
> private const List<ProtoId<DatasetPrototype>> OperationNamesDatasets = new {"operationPrefix", "operationSuffix"};
> ```

You can't have a List const, any type used in a `const` must be a compile-time constant.
https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/constants

Reply:
then what do i do ?

Reply:
just make 2 variables and make the list in the GetRandomFromSegments call